docs/components/related-navigation.md

```diff
@@ -0,0 +1,271 @@
+
+      - <a class="govuk-link" href="https://www.gov.uk/government/statistics/local-authority-collected-waste-for-england-quarterly-estimates" rel="noopener noreferrer" target="_blank">Official statistics (opens in a new tab)</a>
+'
+  6:
+    title: Elsewhere on the web
+    description:
+      # You MUST wrap this in single quotation marks (ie. ' '), since markdown can be used to enter this information. To create a heading, use three hashes (ie. ###).
+      'Appears on, for example:
+
+      - Mainstream smart answers, e.g. <a class="govuk-link" href="https://www.gov.uk/find-a-job" rel="noopener noreferrer" target="_blank">Find a job (opens in a new tab)</a>
+      
+      - Mainstream guides, e.g. <a class="govuk-link" href="https://www.gov.uk/power-of-attorneyn" rel="noopener noreferrer" target="_blank">Make, register or end a lasting power of attorney (opens in a new tab)</a>'
+  7:
+    title: Other contacts
+    description:
+      # You MUST wrap this in single quotation marks (ie. ' '), since markdown can be used to enter this information. To create a heading, use three hashes (ie. ###).
+      'Appears on, for example Mainstream answers, e.g. contact pages like <a class="govuk-link" href="https://www.gov.uk/government/organisations/hm-revenue-customs/contact/employer-paye-payment" rel="noopener noreferrer" target="_blank">Employer PAYE and National Insurance: payments (opens in a new tab)</a> and <a class="govuk-link" href="https://www.gov.uk/government/organisations/hm-revenue-customs/contact/self-assessment" rel="noopener noreferrer" target="_blank">Self Assessment: general enquiries (opens in a new tab)</a>.'
+
+# Evidence and insights for this component
+# List out all past documentation/supporting material with regards to or realted to this component. It can include (1) past design documentation, (2) research findings, and (3) presentations.
+insights:
+  # To add additional insights duplicate the the fields below (adhering to the formating) but increase the count by one integer.
+  0:
+    # A description is REQUIRED in order for this information to render on the page.
+    date: November 2021
+    description:
+      # You MUST wrap this in single quotation marks (ie. ' '), since markdown can be used to enter this information. To create a heading, use three hashes (ie. ###).
+      'This research explored how users who advise others interact with and navigate GOV.UK content page.
+
+
+      Users were found to have varied expectations for what "related content" might mean.
+      The phrase "related content" means different things to users depending on their prior experience of the content and of other resources.
+
+
+      "I do not know if these are maybe ones that have been recently clicked on by someone"
+
+
+      "[related content] might be things that are referred to in that document that if you want more detail on, you click into those pages"
```

Comment:
Why is "related content" wrapped in square brackets?

Reply:
Copied it from the report. I think this is how you insert something into a quote, showing that it is for clarification but not what the personal actually said